so_vector/index.json

```diff
@@ -0,0 +1,36 @@
+{
+  "settings": {
+    "index": {
+      "number_of_shards": 2,
+      "number_of_replicas": 0
+    }
+  },
+  "mappings": {
+    "_source": {
+      "enabled": false
+    },
+    "properties": {
+      "userId": {
```

Comment:
We don't index `body` field?

Reply:
hmm, I meant to index it! It's not used right now, but in the future we'll use it to test hybrid search.

Reply:
Correction: we are indexing it, it just doesn't need to be mapped since it's dynamically mapped correctly as `text`. I'll add a mapping here anyways since it's clearer.